test/functions/generic/genericChannelInInit2.good

```diff
@@ -1,3 +1,5 @@
+genericChannelInInit2.chpl:7: warning: please use '?' when declaring a field with generic type
+genericChannelInInit2.chpl:7: note: for example with 'fileWriter(?)'
```

Comment:
@mppf: Mostly rhetorical question for me, but may be of greater interest to a reviewer:  How did you decide whether to keep a test working by updating its source vs. its .good/.bad file?